### ISSUE TITLE
Frequent lost connections when Blazor Server deployed to Azure ContainerApps

### ISSUE THREAD

**Issue by sravimohan** [Azure/azure-signalr#1512]
### Describe the bug
Frequent connections issues when connecting from a Blazor Server App deployed to the new Azure ContainerApps.  I am using Azure SignalR in standard mode. 

[21:33:15 ERR] Failed to connect to '(Primary)https://****.service.signalr.net', will retry after the back off period. Error detail: The server returned status code '403' when status code '101' was expected.. The server returned status code '403' when status code '101' was expected.. Id: 6197cc98

### To Reproduce
- Setup Azure SignalR in Standard Mode 
- deploy  Blazor server app configured with Azure SignalR  to Azure ContainerApp Service 

### Exceptions (if any)
[21:33:15 ERR] Failed to connect to '(Primary)https://****.service.signalr.net', will retry after the back off period. Error detail: The server returned status code '403' when status code '101' was expected.. The server returned status code '403' when status code '101' was expected.. Id: 6197cc98

### Further technical details
    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="6.0.1" />
    <PackageReference Include="Microsoft.Azure.SignalR" Version="1.13.0" />

**Comment by Y-Sindo** [comment]
Please confirm that your connection string is correct. Additionally, did the issue only occur in the Azure Container Apps or it could be reproduced locally?

**Comment by sravimohan** [comment]
> Please confirm that your connection string is correct.
yes, can confirm. I am using primary connection string. Also it does work intermittently. 

> Additionally, did the issue only occur in the Azure Container Apps or it could be reproduced locally?
yes, I am not seeing the "Failed to connect to" errors in my local logs.

Additional error logs, https://gist.github.com/sravimohan/d43f45f49a21db069944e035f3ecb417

**Comment by Y-Sindo** [comment]
Could you send me your SignalR resource ID so that we could check the log in the back end for you? Email: zityang@microsoft.com

**Comment by sravimohan** [comment]
thank you, I will check out the troubleshooting methods.
I will email the resource id to you.

**Comment by Y-Sindo** [comment]
> yes, can confirm. I am using primary connection string. Also it does work intermittently.

From the logs in the back end, you should be using AAD auth instead of access key. Error code "403" means the app doesn't have the correct role. The strange thing was that 403 only occurred on one IP address. Please confirm all your app instances have the correct role. Hope these information helps you.
